### PR TITLE
Forth commit

### PR DESCRIPTION
**Pull request by techtorial**
Validating the login functionality with the positive scenario.

### CODE REVIEW

File: src/Test.java [hunk 1]
```diff
@@ -5,5 +5,6 @@ public static void main(String[] args) {
         System.out.println("First");
         System.out.println("Second");
         System.out.println("Third");
+        System.out.println("Forth");
```
Comment (Bota1956) [inline src/Test.java]: change

Comment (Bota1956) [inline src/Test.java]: change it please

**Review by Bota1956 — CHANGES_REQUESTED** [review]

File: src/Test.java (hunk 1 above)
Comment (mturanus) [inline src/Test.java]: You should update the misspelling for the fourth.

**Comment by mturanus** [comment]
Great!

**Pull request merged**